clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1744,12 +1790,15 @@ private void internalClose(boolean doFlush) {
         if (opType == ConsumerActionType.RESET_OFFSET) {
           LOGGER.info("Cleanup Reset OffSet. Replica: {}", replica);
           storageMetadataService.clearOffset(topic, partition);
+        } else if (opType == DROP_PARTITION) {
+          PubSubTopicPartition topicPartition = message.getTopicPartition();
```

Comment:
Nit, I don't really know if we need to add a log line here (follow patterns for `RESET_OFFSET`) to indicate the deletion actually happened in the `internalClose`, given that there are already logs in `dropPartitionSynchronously`. Just a heads up, I guess.

Reply:
I think it makes sense to maintain consistency.